logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/user/MarkMLSEnableChangeAsNotifiedUseCase.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.kalium.logic.feature.user
+
+import com.wire.kalium.logic.configuration.UserConfigRepository
+
+/**
+ * Mark the MLS enabling status change as notified
+ * need to be called after notifying the user about the change
+ * e.g. after showing a dialog, or a toast etc.
+ */
+interface MarkMLSEnableChangeAsNotifiedUseCase {
+    suspend operator fun invoke()
+}
+
+class MarkMLSEnableChangeAsNotifiedUseCaseImpl(
+    private val userConfigRepository: UserConfigRepository
+) : MarkMLSEnableChangeAsNotifiedUseCase {
+
+    override suspend fun invoke() {
```

Comment:
Question, do we need to return something to know what to do after, or is it some mechanism, like fire and forget?

Reply:
yeah, i belive: shoot and forget is enough for now